extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -95,7 +105,11 @@ const Track = memo(
 			inlineStyle.color = colors.secondary.custom;
 		}
 
-		/* translators: This needs to be a single word with no spaces. It describes the current item in the group. A screen reader will announce it as "[title], current track" */
+		/*
+		 * Translators: This needs to be a single word with no spaces. It describes
+		 * the current item in the group. A screen reader will announce it as
+		 * "[title], current track".
+		 */
```

Comment:
Have you tested if this comment annotation gets picked up as expected?

Reply:
Fixed in https://github.com/Automattic/jetpack/pull/15281/commits/116bc2b09cbb167da38709b588dc5e4296ed4b5f